### PR TITLE
Update README

### LINKED ISSUE #18: Update README

### PR DESCRIPTION
**Pull request by edelarua**
Based off of the outline used for repos in insightsengineering.

Closes #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,47 @@
 # rlistings
-Value formatting and ASCII rendering infrastructure for tables and listings.
+
+[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
```
Comment (gmbecker) [inline README.md:3]: The rlisting package is not in a stable usable state (ie production). Please change this to whatever the official terminology is for alpha/PoC/similar is

**Review by gmbecker — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (edelarua) [reply README.md]: Changed to WIP. All status options are listed [here](https://www.repostatus.org/).

**Review by Melkiades — APPROVED** [review]
Lgtm! I would also have a very small listing with its code to show at a glance how it is rendered

File: README.md [hunk 2]
```diff
@@ -16,6 +16,23 @@ The `rlistings` R package is a package that was designed to create and display l
...
+
+# Reducing the data
+mtcars_ex <- mtcars %>% mutate("car" = rownames(mtcars)) %>% head(10)
+
+as_listing(mtcars_ex, 
+  key_cols = c("gear", "carb"), 
+  cols = c("gear", "carb", "qsec", "car"))
+  
+```
+
```
Comment (gmbecker) [inline README.md:35]: @Melkiades @edelarua if we're going to have code and output in the README.md file (which I think is a good idea), it needs to be generated via a README.Rmd file (which is how the output will actually get in there). Can one of you please make this change?

**Review by gmbecker — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (edelarua) [reply README.md:96]: I have added an README.Rmd file and regenerated the README.md accordingly.

**Review by Melkiades — APPROVED** [review]
Lgtm! Well done

File: NEWS.md [hunk 3]
```diff
@@ -5,6 +5,8 @@
  * now depends on dplyr instea dof magrittr to hopefully avoid var_labels droppage issues
  * `paginate_listing` now supports pagination in both directions
  * Add Get Started vignette.
+ * Update README.
+
```
Comment (gmbecker) [inline NEWS.md:9]: extraneous newline

**Review by gmbecker — CHANGES_REQUESTED** [review]
Overall looks good, please fix the unnecessary newline in NEWS

**Pull request merged**